### PR TITLE
Extend ui-test, adding local server, test eap, fixes #440

### LINKED ISSUE #440: Include EAP server in the tested servers
Add test covering adding local server (EAP) server.

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

File: src/ui-test/common/util/testUtils.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    }
+}
+
+export async function showErrorNotifications() {
+    const errors = await getNotifications(NotificationType.Error);
+    if (errors && errors.length > 0) {
+        const report = errors.map(async error => {
+            return `${await error.getSource()}: ${await error.getMessage()} \r\n`;
+        });
+        console.log('Error appeared during creating local server adapter: ' + report);
```
Comment (jrichter1) [inline src/ui-test/common/util/testUtils.ts:25]: please use template string for consistency

File: src/ui-test/server/ui/abstractServer.ts [hunk 2]
```diff
@@ -23,46 +23,64 @@ export abstract class AbstractServer implements IServer {
...
+        try {
+            await VSBrowser.instance.driver.wait(async () => await serverStateChanged(this, oldState), 3000);
+        } catch (error) {
+            const menu = await treeItem.openContextMenu();
+            await menu.select(contextMenuItem);
+        }
+        await VSBrowser.instance.driver.wait(
+            async () => await serverHasState(this, expectedState),
+            timeout,
+            'Failed to get expected server state ' + ServerState[expectedState] + ' for ' + await this.getServerName() + ', actual state was ' + ServerState[await this.getServerState()]);
```
Comment (jrichter1) [inline src/ui-test/server/ui/abstractServer.ts:68]: template string pls

File: src/ui-test/server/ui/rspServerProvider.ts [hunk 3]
```diff
@@ -22,52 +22,53 @@ export class RSPServerProvider extends AbstractServer {
...
         await menu.select(AdaptersConstants.RSP_SERVER_PROVIDER_CREATE_NEW_SERVER);
         return await InputBox.create();
     }
 
-    async createNewServerCommand(): Promise<void> {
+    public async createNewServerCommand(): Promise<void> {
         const input = await new Workbench().openCommandPrompt() as InputBox;
-        await input.setText('>' + AdaptersConstants.RSP_COMMAND + ' ' + AdaptersConstants.RSP_SERVER_PROVIDER_CREATE_NEW_SERVER);
+        await input.setText(`> ${AdaptersConstants.RSP_COMMAND} ${AdaptersConstants.RSP_SERVER_PROVIDER_CREATE_NEW_SERVER}`);
         await input.confirm();
```
Comment (jrichter1) [inline src/ui-test/server/ui/rspServerProvider.ts:68]: this is equivalant to calling 
```typescript
await new Workbench().executeCommand(`${AdaptersConstants.RSP_COMMAND} ${AdaptersConstants.RSP_SERVER_PROVIDER_CREATE_NEW_SERVER}`)
```

**Pull request merged**